services/terrain/doc/install.md

```diff
@@ -1,63 +1,63 @@
-# Installing and Configuring Donkey
+# Installing and Configuring Terrain
 
-Donkey is packaged as an RPM and published in iPlant's YUM repositories.  It
-can be installed using `yum install donkey` and upgraded using `yum upgrade
-donkey`.
+Terrain is packaged as an RPM and published in iPlant's YUM repositories.  It
+can be installed using `yum install terrain` and upgraded using `yum upgrade
+terrain`.
```

Comment:
@dennisroberts71 how would you like to handle this? You want me to write up a bug?